### DISCUSSION TITLE
Notification Counts Not Updating after PR is merged

### DISCUSSION THREAD

**Discussion by ChetanTechnource** [community/community#28137] [Pull Requests]
Hello Team, 

I have been noticing  an issue in Pull Requests tab. Once you visit that page, there will be number of pending PR count on top. But when the PR is merged that count remains as it is.

You need to refresh the page to reflect actual pending PRs.

Am I the only one facing this issue ?

Kindly check.

Thank you.

**Comment by jfullstackdev** [comment]
PR not yet merge, there is one count

![image](https://user-images.githubusercontent.com/47092464/182581127-fe179a4f-a5f0-4907-9619-c275a1d1713a.png)

then PR merged, the count is gone without reloading

![image](https://user-images.githubusercontent.com/47092464/182581307-8cc1bb02-4340-4ced-a59f-db3207549557.png)
